addons/mail/static/src/core/web/recipients_input.js

```diff
@@ -193,23 +173,35 @@ export class RecipientsInput extends Component {
     getTagsFromMailThread() {
         const tags = [];
         const createTagForRecipient = (recipient, recipientField) => {
-            const title = _t("%(partner_name)s <%(partner_email)s>", {
-                partner_name: recipient.name,
-                partner_email: recipient.email,
-            });
+            const title = _t(
+                recipient.email ? "%(partner_name)s <%(partner_email)s>" : "%(partner_name)s",
```

Comment:
This makes the value non-translatable, only string are allowed inside `_t`

Please provide 2 different translation terms if there is a conditional rendering.